### ISSUE TITLE
Add image and video lightbox on part-up startpage

### ISSUE THREAD

**Issue by ralphboeije** [part-up/part-up#1437]
Originally this issues came from the part-up startpage feature, but I think it also applies to the personal homepage as we also render documents and images on updates there. This light box feature is not yet part of the react app.

On click on image/video in media gallery, a lightbox should open to show the image and be able to play video. 
- add download button for image to lightbox

**Comment by ralphboeije** [comment]
@noelheesen Leon didn't have time for this one today. This a Blaze part, so could you taken on this one?

**Comment by noelheesen** [comment]
When you click an image attached to a message or activity a gallery opens. Is this what you mean? Else I will implement via http://lokeshdhakar.com/projects/lightbox2/

**Comment by ralphboeije** [comment]
@noelheesen Yes, that is what I mean. As the left column on the part-up is in blaze of course, this issue actually consists of two:
1. Add on click handler to image gallery of part-ups (on top of in left column) and have it open the gallery that is used already to display images attached to messages or activities. AND add a download button next to the close button on this gallery/lightbox
2. Implement this feature in react to be used on the personal homepage and the startpage (on click on an image in a message or activity there)

**Comment by noelheesen** [comment]
The current blaze implementation is complex and it can be simplified by using a library like lightbox2.
Complexity: 15~
